### PR TITLE
[backend] add description for sample docker image

### PR DESCRIPTION
**Pull request by IJtLJZ8Rm4Yr**

### CODE REVIEW

File: ymir/backend/src/ymir_app/app/config.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ class Settings(BaseSettings):
 
     # RUNTIME
     RUNTIMES: Optional[
-        str] = '[{"name": "sample_image", "hash": "6d30c27861c5", "type": 1, "url": "industryessentials/executor-example:latest", "configs": [{"expected_map": 0.983, "idle_seconds": 60, "trigger_crash": 0, "type": 1}, {"idle_seconds": 6, "trigger_crash": 0, "type": 2}, {"idle_seconds": 3, "trigger_crash": 0, "type": 9}]}]'  # noqa: E501
+        str
+    ] = '[{"name": "sample_image", "hash": "6d30c27861c5", "description": "demonstration only", "url": "industryessentials/executor-example:latest", "configs": [{"expected_map": 0.983, "idle_seconds": 60, "trigger_crash": 0, "type": 1}, {"idle_seconds": 6, "trigger_crash": 0, "type": 2}, {"idle_seconds": 3, "trigger_crash": 0, "type": 9}]}]'  # noqa: E501
```
Comment (phoenix-xhuang) [inline ymir/backend/src/ymir_app/app/config.py]: Demonstration only. This docker image trains fake model after a requested length of running time period. These models can only be used by this same docker image for re-training/mining/inference purposes. Adjust the hyper-parameters to set the length of running time, or to trigger crash, to set expected mAP, etc.

Comment (IJtLJZ8Rm4Yr) [reply ymir/backend/src/ymir_app/app/config.py]: len(desc) < 100

Comment (phoenix-xhuang) [reply ymir/backend/src/ymir_app/app/config.py]: 可以增加长度吗？

Comment (IJtLJZ8Rm4Yr) [reply ymir/backend/src/ymir_app/app/config.py]: 要 migrate table

Comment (phoenix-xhuang) [reply ymir/backend/src/ymir_app/app/config.py]: 可以，下次你要修改table的时候一起改吧

Comment (IJtLJZ8Rm4Yr) [reply ymir/backend/src/ymir_app/app/config.py]: updated

**Review by phoenix-xhuang — APPROVED** [review]

**Review by fenrir-z — APPROVED** [review]

**Pull request merged**